docs/cues/cue-playback.md

```diff
@@ -176,42 +176,64 @@ Settings](playback-options.md#fader-tab) in playback [Options](playback-options.
 See [Speed and Size Masters](../running-the-show/playback-controls.md#speed-and-size-masters)
 for more details.
 
-Releasing Running Playbacks
+Release
 ---------------------------
 
-You can **release** a running playback to its previous state by pressing
-\<Release\> *(\<Off\> button on Pearl Expert)*, then the **Select** button of
-the playback to be released. Channels will release back to their state
-in the previous playback until no playbacks are left to be released. If
-the \[Release to Home\] setting on the Release menu is Enabled, channels
-will go to the power-on state, which you can program (see 
-[next section](#programming-the-release-power-on-state)).
-If the option is disabled, channels will remain in the state of the last
-playback. You can release all running playbacks by pressing \<Release\>
-then \[Release all playbacks\].
-
-You can also configure the key profile so that the blue or grey buttons
+Normally when you kill a playback by lowering the fader, the LTP attributes will be left in the last state set by the playback. Sometimes you don't want this, for example if you have a strobe cue, you don't want the strobing to carry on when you kill the cue. The **Release** function lets you set how attributes will return to their previous values.
+
+You can set a mask so that some attributes will automatically release from the playback when the playback is killed - see [Release Mask](#release-mask) below. For a cue this will happen when all fade times have completed.
+
+Attributes will go back to the state set in the most recent previous playback until no playbacks are left to be released. If the \[Release to Home\] setting on the Release menu is Enabled, channels will go to the power-on state, which you can program (see [Programming the Release / Power On State](#programming-the-release-power-on-state)).
+If the option is disabled, channels will remain in the state of the last playback.
+
+You can also configure the key profile so that the handle buttons 
 can be used to release the playback using [Key Profiles](../system-settings/key-profiles.md).
 
-Channels will always release with a fade time, this defaults to 2
-seconds but you can change it using \[Master Release Time\] in the
-Release menu. You can also set an individual release time for the
-playback from the [Release tab](playback-options.md#release-tab) of playback [Options](playback-options.md).
+Channels will always release with a fade time, this defaults to 2 seconds but you can change it using \[Master Release Time\] in the Release menu. You can also set an individual release time for the playback from the [Release tab](playback-options.md#release-tab) of playback options.
 
--   You can instantly kill a playback by pressing \<Avo\> + the **Select**
-    button of the playback.
+-   You can instantly kill a playback (ignoring any fade out times) by pressing \<Avo\> + the **Select**
```

Comment:
So for a memory, I believe Avo+Select will set the playback level (the "fader level") to zero, with the memory reacting in the same way as if you had manually moved the fader to zero; so a mode1/2/3 memory with fade out times will still fade over its fade out time.

Reply:
So is that the same as doing \<Release\> + Select, apart from the Avo button method would use the mask programmed for the playback and the Release method would use a temporary mask?

Reply:
OK it's the opposite way round to what I put. Release+Select kills/releases instantly. Avo+Select kills after fade out.

Reply:
Hmm, so I would expect Release + Select to kill/release over the master (or override) release time, not instantly?

Reply:
Umm, well that is a description problem. It kills instantly ignoring any fade times, but then releases over the release time, I'll add that in